Wall-E/Assets/Scripts/LayerController.cs

```diff
@@ -3,37 +3,13 @@
 using UnityEngine;
 
 public class LayerController : MonoBehaviour {
-	public GameObject player;
-
-	private const string SORTING_LAYER_BACKGROUND = "Environment";
-	private const string SORTING_LAYER_FOREGROUND = "Foreground";
-	private const int SORTING_ORDER = 1;
-
-	private SpriteRenderer objectSprite;
-
-	private Vector3 playerPosition;
-	private Vector3 objectPosition;
 
 	void Start() {
-		objectSprite = GetComponent<SpriteRenderer>();
 	}
 
 	// Update is called once per frame
 	void Update () {
-		playerPosition = player.transform.position;
-		objectPosition = transform.position;
-		Debug.Log(playerPosition.y);
-		Debug.Log(objectPosition.y);
-
-		DynamicLayerer();
+			GetComponent<SpriteRenderer>().sortingOrder = Mathf.RoundToInt(transform.position.y*100)*-1;
```

Comment:
whilst it's lovely to have this on one line, might be clearer long term to have the Sprite Renderer component stored in a variable incase it needs to be used elsewhere in the future but up to you!

Reply:
I dont really expect we'll need to use any of these objects again in this script, so it's probably fine not to, right? or are you suggesting making it public or something so other scripts can use it?